### PR TITLE
Casino update

### PR DESCRIPTION
**Pull request by SeeminglyTypicalUsername**
## About The Pull Request

This PR  improves upon the casinos with new carpets, roulette tables, and a sign for the shark club which is animated.
image
image

## Why It's Good For The Game

The casinos needed to be further individualized. 

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: shark club sign
add:  multiple new carpets
add:  mordino armory
/:cl:

### CODE REVIEW

**Review by hermaplusplus — CHANGES_REQUESTED** [review]
Eye gouging is not a map feature, and should be in a separate pull request targeting the main branch. Please do not bundle game mechanics with map changes.

This PR also includes the addition of a roulette table in code, which is already present in PR #40 targeting `main`. Please remove it from this PR, as it will be in the main code soon. You should be fine to leave the map files as they are.

**Review by hermaplusplus — APPROVED** [review]

**Comment by hermaplusplus** [comment]
**Note to maintainers**: Before this PR is merged, #40 must be merged and `main` must be merged into `maps/newreno`. In lieu of CI on the mapping branches, test for runtimes thoroughly.

**Review by Kitsunemitsu — APPROVED** [review]
Took a look, Everything looks good.
We really do need to get MapDiffBot for easier map PR reviews

**Pull request merged**
